railties/lib/rails/generators/app_base.rb

```diff
@@ -340,7 +340,15 @@ def spring_install?
       end
 
       def run_bundle
-        bundle_command('install') if bundle_install?
+        # Most of us have the CPUs; so let's parallelize `bundle install`
```

Comment:
I don't think we need any comment. This seems more to be describing/justifying the change, rather than the resulting code.

We can introduce a new method, though, instead of doing the work in line here.

... which could also fix the fact we're now doing rather a lot of work before we check `bundle_install?`.